modules/openapi-generator/src/main/resources/go/client.mustache

```diff
@@ -419,6 +419,15 @@ func (c *APIClient) decode(v interface{}, b []byte, contentType string) (err err
 		*s = string(b)
 		return nil
 	}
+	if f, ok := v.(**os.File); ok {
+		*f, err = ioutil.TempFile("", "HttpClientFile")
```

Comment:
I got the following errors when testing the petstore go client: 
```
go-petstore/client.go:391:13: undefined: ioutil
```

Can you please take a look when you've time?

Reply:
Sorry about that. That is my mistake for missing that as my dev environment will do auto-imports that hid this.

Reply:
Looks like you merge another PR into your PR. 

Can you file a PR with just the fix for downloading the files?